convenor/regTeam.php

```diff
@@ -250,6 +156,36 @@ function test_input($data) {
     <!-- /#wrapper -->
 
     <?php include '../global/jqueryref.php'; ?>
+
+    <script type='text/javascript'>
+        function addFields(){
+            // Number of inputs to create
+            var number = document.getElementById("team_Size").value;

```

Comment:
Why do you prefer this? Willing to make changes if you provide a compelling argument

Reply:
Just realised its an ES6 thing so we wouldn't even be able to implement. But `const` and `let` are way safer because they're less ambiguous. `const` will definitely never be reassigned, `let` can only be used in the scope of the block rather than the function.